packages/frontend/pages/lobby.tsx

```diff
@@ -1,23 +1,35 @@
-import { ReactElement } from 'react';
+import { ReactElement, useEffect } from 'react';
```

Comment:
useEffect into _app.tsx maybe? That way it'll always on any page that we are on.